collections/copernicus-dem.yaml

```diff
@@ -95,22 +95,37 @@ CubeDimensions:
     extent:
       - -180
       - 180
+    reference_system: 4326
   y:
     type: spatial
     axis: y
     extent:
       - -85
       - 85
+    reference_system: 4326
   t:
     type: temporal
     extent:
       - '2010-12-12T05:04:33Z'
       - '2015-01-16T01:37:12Z'
-    step: P5D
   bands:
     type: bands
     values:
       - DEM
       - dataMask
+Summaries:
+  raster:bands:
+    - description: Heights in meters
+      name: DEM
+      openeo:gsd:
+        value:
+          - 90
+          - 90
+        unit: m
+        # https://docs.sentinel-hub.com/api/latest/data/dem/#mission-information
```

Comment:
Did you check locally that this works? I think json doesn't support comments.

Reply:
I planned to remove the comments after we decide which values to use (and change them to degrees) / before this PR will be merged.

Reply:
Ran `npm run build` and the comments are not present in the generated stac json files, so I think it's all good even if they stay. But I would still rather remove them before merging.

Reply:
Following the ESA documentation the cop-30 has a grid spacing of 1.0'' = this equals 0.000277777777777° (at the equator)  which I think we can/should use here (also for mapzen dem). I know it's not 100% accurate but Vito need some number here and that's the best we can specify it.

Agree that we can remove the comments then. I will add the decision to us the best available resolution (cop30 for this collection) to our documentation.

Reply:
Great, thanks!